packages/atomic/src/components/search/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -63,14 +64,32 @@ export class AtomicResultText implements InitializableComponent {
     }
   }
 
+  private possiblyWarnOnBadFieldType() {
+    const resultValueRaw = ResultTemplatesHelpers.getResultProperty(
+      this.result,
+      this.field
+    );
+    if (isArray(resultValueRaw)) {
+      this.bindings.engine.logger.error(
```

Comment:
Instead of logging the error in the logger like this, I think we should leverage `this.error` just like `atomic-result-date` does. We could move the validation to `componentWillRender`.

Reply:
In this particular case, I'm not sure that rendering a end user error is a good idea, since this is inside a result template.